src/style/_legacy/components/productions/credits_panel.scss

```diff
@@ -1,26 +1,3 @@
-.credits_panel {
```

Comment:
Removing this implies moving the layout of these types should include having icons like results.